jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/util/Select.java

```diff
@@ -52,10 +53,18 @@ public void addOption(final String text, final String value) {
     }
 
     public void addOption(final String text, final String value, final Boolean selected) {
+        addOption(text, null, value, selected);
+    }
+
+    public void addOption(final String text, final String subText, final String value, final Boolean selected) {
```

Comment:
Do we need to expose the boolean argument?
It seems that in usages of all overloaded methods, only _false_ value is used.